.github/workflows/bump-versions.yaml

```diff
@@ -0,0 +1,112 @@
+name: Bump chart and/or app versions
+permissions:
+  contents: write
+  pull-requests: write
+on:
+  workflow_call:
+    inputs:
+      bypass-dev-protections:
+        description: >
+          Use custom user to override branch protections when
+          pushing directly to dev
+        required: false
+        default: false
+        type: boolean
+    secrets:
+      PUSH_CHARTS_TOKEN:
+        required: false
+
+jobs:
+  # Increment the APP and CHART versions based on commit message starting with Update-versions 
+  bump-versions:
+    name: Get supported Go versions
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Calculate desired version changes
+        id: get-new-versions
+        run: |
+          . version.env
+          outputs=""
+          for commit in ${{ github.event.push.commits }}; do
```

Comment:
What does this loop do? Is this checking more than the merge commit?

Reply:
Yes, it checks all commits in the PR for now